modules/server/src/network/gql.ts

```diff
@@ -1,13 +1,26 @@
 import axios from 'axios';
-import { print } from 'graphql';
+import { ASTNode, print } from 'graphql';
 
-export const createGqlQuery = ({ query: queryAST, variables }) => {
+/**
+ * Creates a graphql query string with variables for use in a POST request
+ * @returns string
+ */
+export const createGqlQuery = ({
```

Comment:
Noticed this function isn't actually used in your code here, is that expected?

Reply:
expected. https://github.com/overture-stack/arranger/pull/877#discussion_r1635117424